eng/helix/helix.proj

```diff
@@ -74,7 +74,7 @@
     <HelixProperties Condition="'$(RunQuarantinedTests)' != 'true'" Include="runType" Value="unquarantined" />
   </ItemGroup>
 
-  <Target Name="IncludeAspNetRuntime" BeforeTargets="Gather">
+  <Target Name="IncludeAspNetRuntime" BeforeTargets="Gather" Condition="'$(DoNotIncludeSharedFxHelix)' != 'true'">
```

Comment:
It's up to you but I was thinking of something more like
```suggestion
  <Target Name="IncludeAspNetRuntime" BeforeTargets="Gather"
      Condition="'$(DoNotIRequireSharedFxHelix)' != 'true' OR
        EXISTS('$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\Microsoft.AspNetCore.App.Runtime.$(TargetRuntimeIdentifier).$(SharedFxVersion).nupkg')">
```
Then `RunHelix.ps1` wouldn't need a new `[switch]` and could pass `/p:DoNotIRequireSharedFxHelix=true` unconditionally.

Reply:
Separately, @HaoK could you remind me why the `EXISTS('...')` exists on line 90 of this file❔ We should avoid ordering issues and ensure the shared Fx packages exist before creating the correlation payload because it's created _once_ for all projects under test. If the order isn't working right, tests that require the shared Fx e.g. ProjectTemplates.Tests.csproj will fail annoyingly late.

Reply:
My approach was sort of the backwards of what you are implying, I wanted to add targets that no-op or didn't fail since I believe things have morphed/transformed from how the targets were originally named, before we had a single option which included the runtime and did a lot of things, so lots of tests set the flag but they might not be building the shared Fx since only templates needed them.  If we have more granular options now, its likely the exists is redundant now if this is only for the shared fx

Reply:
The `EXISTS` isn't redundant for Microsoft.AspNetCore.App.Ref because we hope not to service that package but @BrennanConroy please remove the one for Microsoft.AspNetCore.App.Runtime if you choose to update this PR again.